docs/user/data_collection.rst

```diff
@@ -83,3 +83,106 @@ reported under the key ``shield-recipe-client/recipe/<recipe id>``:
 .. js:data:: RECIPE_SUCCESS
 
    The recipe was executed successfully.
+
+
+Enrollment
+-----------
+Shield records enrollment and unenrollment of users into studies, and
+records that data using `Telemetry Events`_. All data is stored in the
+``normandy`` category.
+
+.. _Telemetry Events: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/events.html
+
+Preference Studies
+^^^^^^^^^^^^^^^^^^
+Enrollment
+   method
+      The string ``"enroll"``
+   object
+      The string ``"preference-study"``
+   value
+      The name of the study (``recipe.arguments.slug``).
+   extra
+      branch
+         The name of the branch the user was assigned to (example:
+         ``"control"`` or ``"experiment"``).
+      experimentType
+         The type of preference experiment. Currently this can take
+         values "exp" and "exp-highpop", the latter being for
+         experiments targetting large numbers of users.
+
+Unenrollment
+   method
+      The string ``"unenroll"``.
+   object
+      The string ``"preference-study"``.
+   value
+      The name of the study (``recipe.arguments.slug``).
+   extra
+      resetValue
+         The string ``"true"`` if the preference was set back to its
+         original value, ``"false"`` if it was left as its current
```

Comment:
Out of curiosity, why strings that contain booleans? Why not just booleans?

Also, in the epitome of nitpicks, the name of this property makes me think its value is the value the preference was reset to, not whether or not it was reset. Dunno if there are obvious better choices. `didResetValue` or something?

Reply:
According to [the docs](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/events.html#recordevent), API only allows "string keys & values", so booleans aren't an option.

I think you're right about the name of `resetValue`. There isn't a lot of outside-of-the-code history to this name, so now is a good time to change it.